services/merak-topo/handler/handler.go

```diff
@@ -203,24 +293,22 @@ func UpdateComputenodeInfo(client *kubernetes.Clientset, topo_id string, returnM
 	// log.Printf("updatecomputenode:=========Update mac addresses ===========")
 
 	for _, node := range topo.Vnodes {
-		var cnode pb_common.InternalComputeInfo
```

Comment:
@zzengcs so it seems like every time the `UpdateComputenodeInfo` got called, it will run through 10k vnodes? If it got called many time, then it will cost n*10k.

Reply:
Save this part for future development

Reply:
@zzengcs please create an issue for this problem. thanks.